docs/custom-registry-id.md

```diff
@@ -0,0 +1,24 @@
+# How to use a custom regisry ID
+
+Azure docker registries has a typical ID of the format `*-microsoft.azurecr.io`. A customer might like to have a custom registry ID that associate with its own organization. The following is the guide on how to achieve that.
+
+## Prerequisites
+
+To setup custom registry ID, the following prerequisites are needed:
+
+* A front end docker host whichs hosts the desired regisry ID. We will run an nginx docker image on it. In this guide we would use the example: `registry.azurecr-test.io`
```

Comment:
>whichs [](start = 26, length = 6)

Please try this out - [https://github.com/Microsoft/vscode-spell-check](https://github.com/Microsoft/vscode-spell-check)